### PR TITLE
chore: add feature flag for request counting inside Unleash

### PR DESCRIPTION
**Pull request by daveleek**
## About the changes

Adds the feature flag for use in request traffic counting

### CODE REVIEW

File: src/lib/types/experimental.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export type IFlagKey =
     | 'createdByUserIdDataMigration'
     | 'showInactiveUsers'
     | 'inMemoryScheduledChangeRequests'
+    | 'trafficDataUsage'
```
Comment (gastonfournier) [inline src/lib/types/experimental.ts:51]: Maybe "collect traffic data" or "collect traffic data in unleash"?

**Review by gastonfournier — APPROVED** [review]
Just details

File: src/lib/types/experimental.ts (hunk 1 above)
Comment (daveleek) [reply src/lib/types/experimental.ts:51]: It's also going to control the UI for data usage, does that impact how you feel about the name?

Comment (daveleek) [reply src/lib/types/experimental.ts]: as discussed I've changed it to `collectTrafficDataUsage`

**Pull request merged**